client/my-sites/site-settings/style.scss

```diff
@@ -625,3 +625,38 @@ button.site-settings__general-settings-set-guessed-timezone.button.is-borderless
 	margin-top: -16px;
 	margin-bottom: 16px;
 }
+
+.site-settings__advanced-customization-notice {
+	display: flex;
+	padding: 20px;
+	background-color: #f4f4f4;
+	align-items: center;
+	justify-content: space-between;
+
+	.site-settings__advanced-customization-notice-cta {
+		display: flex;
+		align-items: center;
+	}
+
+	svg.gridicon.gridicons-info-outline {
+		color: #d38d16;
```

Comment:
I think this should use a variable from the color scheme: https://github.com/Automattic/wp-calypso/blob/trunk/packages/calypso-color-schemes/src/shared/color-schemes/_default.scss

Maybe `--color-warning-30`?

Reply:
Changed, thanks for pointing to the file!